### PR TITLE
Created md (Moldova) folder, added imageries and their properties

### PR DESCRIPTION
**Pull request by Oni-DOS**
These are the basical imagery used by Moldovan OSM Community. The Moldovan Agency for Land Relations and Cadaster of the Republic of Moldova permit using these imagery in OSM and other non-profit projects. They work in accordance with the law on the national spatial data infrastructure ( https://www.legis.md/cautare/getResults?doc_id=96633 ) and with the Regulation on the rules for the sharing of spatial data sets and related services between public entities and third parties ( https://www.legis.md/cautare/getResults?doc_id=102213 )

### CODE REVIEW

**Comment by grischard** [comment]
First of all, congratulations to the Moldova community! It's always special when you get to create your folder in ELI :). I also see that this is your first pull request ever, so double congratulations.

The `:` in your source id is causing the CI to fail. The acceptable regex pattern is `^[-_.A-Za-z0-9]+$`.

**Comment by Oni-DOS** [comment]
Hello! And thanks for your respond! I fixed the regex errors but I don't understand what is "ERROR:root:Exception: 'NoneType' object has no attribute 'findall'".

**Comment by rbuffat** [comment]
> Hello! And thanks for your respond! I fixed the regex errors but I don't understand what is "ERROR:root:Exception: 'NoneType' object has no attribute 'findall'".

@Oni-DOS The error comes from the library checking WMTS sources. I agree this is not a helpful error message.  But I guess you meant to specify the type as `"type": "wms"`and not `"type": "wmts"` for Ortophoto_2016_RM_Reserve-Server.geojson.

File: sources/europe/md/Ortophoto_2016_RM_Reserve-Server.geojson [hunk 1]
```diff
@@ -0,0 +1,1741 @@
+{
+    "type": "Feature",
+    "properties": {
+        "description": "2016 Ortophotos, Moldova, Scale 1:5000, 20cm, true colours, covers practically the entire territory of the Republic of Moldova, except some areas from the east part of Moldova. [Geoportal.md server]",
+        "id": "Geoportal-Ortofoto_2016_RM",
+        "name": "Moldova Ortophoto (2016) / Geoportal.md Reserve Server",
+        "type": "wmts",
```
Comment (rbuffat) [inline sources/europe/md/Ortophoto_2016_RM_Reserve-Server.geojson]: I assume this should be `"type": "wms"`

Comment (Oni-DOS) [reply sources/europe/md/Ortophoto_2016_RM_Reserve-Server.geojson]: Ah! Yeah, that's true, here should to be wms.

Comment (Oni-DOS) [reply sources/europe/md/Ortophoto_2016_RM_Reserve-Server.geojson]: Thanks for the help!

**Comment by grischard** [comment]
LGTM. @rbuffat?

**Comment by grischard** [comment]
No complaints means we can merge :). Congratulations again. The layers will be in the next version of iD. For the other editors, don't forget to add the layers to the JOSM wiki.

**Comment by Oni-DOS** [comment]
Thanks! In next day I will try to add these layers and to JOSM wiki. In the next weeks I will add and others old imagery (from 2014, 2011, 2007 and 2006). Thanks again for the help!

**Comment by rbuffat** [comment]
@grischard  I had not yet time to look at the license terms, but you have way more experience than me, so you don't have to wait for my reaction!

**Pull request merged**